app/root.tsx

```diff
@@ -91,12 +92,5 @@ interface ErrorBoundaryProps {
 }
 
 export function ErrorBoundary({ feil }: ErrorBoundaryProps) {
```

Comment:
```suggestion
export function ErrorBoundary() {
```

Siden man ikke bruker error-objektet til noe kan vi fjerne det. Da kan vi også fjerne `ErrorBoundaryProps`

Reply:
Santsant. Endret her: [6f9a851](https://github.com/bekk/nav-familie-endringsmelding/pull/45/commits/6f9a85197914fcae58040733862a5ddb0ca16c10)